### PR TITLE
improve readme

### PR DESCRIPTION
**Pull request by diegomarvid**
In order to check the readme in the Repository, you can go to this page https://github.com/google/temporian/tree/readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,65 @@
...
+
+## Supported Features
+Temporian currently supports the following features for pre-processing your temporal data:
+
+* **Simple Moving Average:** calculates the average value of each feature over a specified time window.
+* **Lag:** creates new features by shifting the time series data backward in time by a specified number of time steps.
+* **Leak:** creates new features by shifting the time series data forward in time by a specified number of time steps.
+* **Arithmetic Operations:** allows you to perform arithmetic operations (such as addition, subtraction, multiplication, and division) on time series data, between different events.
+
+More features comming soon...
```
Comment (ianspektor) [inline README.md:59]:
```suggestion
More features coming soon!
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,65 @@
...
+
+
+## Usage Example
+
+A minimal end-to-end run looks as follows:
+
+```python
+import temporian as t
+
+# Load the data.
```
Comment (ianspektor) [inline README.md:16]: Remove fullstop after comment since no comments below have it

File: README.md [hunk 3]
```diff
@@ -1,13 +1,65 @@
...
+    input=event,
+    lag=t.week(1),
+)
+
+# Assign features
+output_event = t.assign(event, sma)
+output_event = t.assign(output_event, lag)
+
+
+# Execute pre processing functions and get results
```
Comment (ianspektor) [inline README.md:39]:
```suggestion
# Execute pipeline and get results
```

File: README.md [hunk 4]
```diff
@@ -1,13 +1,65 @@
...
+        event: event_data,
+    },
+)
+
+```
+
+>__Warning__: The library is still under construction. This example usage is what we are aiming to build in the short term.
+
+## Supported Features
+Temporian currently supports the following features for pre-processing your temporal data:
```
Comment (ianspektor) [inline README.md:52]: Some formatting of the markdown file seems off, `prettier` or a similar vscode extension should fix that for you

File: README.md [hunk 5]
```diff
@@ -1,13 +1,65 @@
...
+
+* **Simple Moving Average:** calculates the average value of each feature over a specified time window.
+* **Lag:** creates new features by shifting the time series data backward in time by a specified number of time steps.
```
Comment (ianspektor) [inline README.md:55]:
```suggestion
* **Lag:** creates new features by shifting the time series data backwards in time by a specified period.
```

File: README.md [hunk 6]
```diff
@@ -1,13 +1,65 @@
...
+* **Leak:** creates new features by shifting the time series data forward in time by a specified number of time steps.
```
Comment (ianspektor) [inline README.md:56]:
```suggestion
* **Leak:** creates new features by shifting the time series data forward in time by a specified period.
```

File: README.md [hunk 7]
```diff
@@ -1,13 +1,65 @@
...
+* **Arithmetic Operations:** allows you to perform arithmetic operations (such as addition, subtraction, multiplication, and division) on time series data, between different events.
+
+More features comming soon...
+
+
+## Requirements for Contributors
```
Comment (ianspektor) [inline README.md:62]:
```suggestion
## Environment Setup
```

File: README.md [hunk 8]
```diff
@@ -53,7 +105,7 @@ Finally, activate the virtual environment by executing:
 poetry shell
 ```
 
-## Run all tests
+## Testing for Contributors
```
Comment (ianspektor) [inline README.md:108]:
```suggestion
## Testing
```

File: README.md [hunk 9]
```diff
@@ -1,13 +1,65 @@
...
+A minimal end-to-end run looks as follows:
+
+```python
+import temporian as t
+
+# Load the data.
+event_data = t.read_event("path/to/data.csv")
+
+# Event schema
+event = t.Event([t.Feature("sales", int), t.Feature("costs", int)])
```
Comment (ianspektor) [inline README.md:18]:
```suggestion
event_data = t.read_event("path/to/data.csv")
event = event_data.schema()
```

File: README.md [hunk 10]
```diff
@@ -1,13 +1,59 @@
...
-![formatting](https://github.com/google/temporian/actions/workflows/formatting.yaml/badge.svg)
+**Temporian** is a library to pre-process temporal signals before their use as input features with off-the-shelf tabular machine learning libraries (e.g., TensorFlow Decision Forests, scikit-learn).
 
-## Requirements
+## Usage Example
+
+A minimal end-to-end run looks as follows:
+
+```python
+import temporian as t
```
Comment (achoum) [inline README.md:12]: Update

**Review by rstz — APPROVED** [review]
Thank you!

**Review by ianspektor — APPROVED** [review]

**Pull request merged**